src/views/v2/month/day.php

```diff
@@ -14,7 +14,8 @@
  */
 // @todo: This is just for presentation purposes, while building the view.
 $day_number   = ( $week < 1 ) ? ( $day + 1 ) * ( $week + 1 ) : ( $day + 1 ) + $week * 7;
-$month_number = 5;
+$month_number = 6;
+$month = $this->get( 'month' );
```

Comment:
Use extracted `$month` var here?

Reply:
Maybe it's a bit confusing, but `$month` here, is the array containing the month data structure (with events on each day). It's hooked from `month-view-demo-template-tags.php` where the function with long names live.